app/views/compute_resources_vms/form/vmware/_base.html.erb

```diff
@@ -18,6 +18,12 @@
 <% checked = params[:host] && params[:host][:compute_attributes] && params[:host][:compute_attributes][:start] || '1' %>
 <%= checkbox_f f, :start, { :checked => (checked == '1'), :help_inline => _("Power ON this machine"), :label => _('Start'), :label_size => "col-md-3" } if new_host and controller_name != 'compute_attributes' %>
 
+<% checked = params[:host] && params[:host][:compute_attributes] && params[:host][:compute_attributes][:memoryHotAddEnabled] || '0' %>
+<%= checkbox_f f, :memoryHotAddEnabled, { :checked => (checked == '1'), :help_inline => _("Enable memory hot add"), :label => _('Memory hot add'), :label_size => "col-md-2" } if new_host and controller_name != 'compute_attributes' %>
```

Comment:
And lastly, I think you might want to move them above the Power ON line, up with the other hardware settings.